### ISSUE TITLE
Invalid Argument - Using Dask

### ISSUE THREAD

**Issue by Sohambutala** [OSOceanAcoustics/echoflow#18]
Ran into an issue while using dask client gather where the code worked perfectly fine when executing it sequentially. Initially thought it might be something related to IO access since we are dealing with local files, but doesn't look like the case since switching to prefect-dask worked fine.

**Comment by lsetiawan** [comment]
Is this related to https://github.com/OSOceanAcoustics/echoflow/issues/17? Does it still occur now?

**Comment by Sohambutala** [comment]
Yes, this is related to #17. I have changed the code to use prefect-dask now (which was working). I will give it a try using dask and get back to you!

**Comment by Sohambutala** [comment]
Closing this issue, since prefect-dask is the preferred and more appropriate way to use dask. Current workflow works with prefect-dask!
